website/components/Footer.tsx

```diff
@@ -142,50 +142,36 @@ export function Footer() {
 }
 
 function SubmitForm() {
-  const [email, setEmail] = useState("");
   const router = useRouter();
 
-  const [message, setMessage] = useState("");
-
-  const subscribe = async (e: FormEvent) => {
+  const handleSubmit = (e: FormEvent) => {
     e.preventDefault();
-    // 3. Send a request to our API with the user's email address.
-    const res = await fetch(`/api/subscribe?email=${encodeURI(email)}`);
-
-    const { error } = await res.json();
-
-    if (error) {
-      // 4. If there was an error, update the message in state.
-      setMessage(error);
-
-      return;
-    }
-
     router.push("/confirm");
   };
 
   return (
     <form
       className="mt-4 sm:flex sm:max-w-md"
       autoComplete="off"
-      onSubmit={subscribe}
+      action="https://cloudquery.us1.list-manage.com/subscribe/post"
+      method="POST"
+      onSubmit={handleSubmit}
     >
       <label htmlFor="email-address" className="sr-only">
         Email address
       </label>
       <div>
+        <input type="hidden" name="u" value="5a433922a8db867a38c8a4a8e" />
```

Comment:
Taken by following https://mailchimp.com/help/host-your-own-signup-forms/ (explained in the issue0